src/SourceLocator/Type/AutoloadSourceLocator.php

```diff
@@ -183,31 +183,30 @@ public function stream_open($path, $mode, $options, &$opened_path) : bool
      * Must be implemented to return some data so that calls like is_file will work.
      *
      * @param string $path
-     * @param int    $flags
+     * @param int    $flags|false
      * @return mixed[]
      * @see https://php.net/manual/en/class.streamwrapper.php
      * @see https://php.net/manual/en/streamwrapper.url-stat.php
      * @phpcsSuppress SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingParameterTypeHint
      */
-    public function url_stat($path, $flags) : array
+    public function url_stat($path, $flags)
```

Comment:
The `array` return type was removed, but the documented type is still `mixed[]`: what's going on here?

Reply:
Fixed